graphql-dynamodb-manager-core/src/main/java/com/fleetpin/graphql/dynamodb/manager/table/Table.java

```diff
@@ -39,10 +39,10 @@ public void setId(String id) {
 		this.id = id;
 	}
 	
-	final void setCreatedAt(Instant createdAt) {
+	public void setCreatedAt(Instant createdAt) {
```

Comment:
these where final and non public and package visibility so that things that implement this can't set these